.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+repos:
```

Comment:
I'm guessing that this is the critical file, with the rest being `black` doing its worst?